platform/ios/src/MGLMapView.h

```diff
@@ -610,7 +610,7 @@ MGL_EXPORT IB_DESIGNABLE
  *
  * The default minimumZoomLevel is 0.
  */
-@property (nonatomic) IBInspectable double minimumZoomLevel;
+@property (nonatomic) double minimumZoomLevel;
```

Comment:
Would you mind making the corresponding changes to the macOS implementation of MGLMapView? Thanks!